lib/bolt/bolt_option_parser.rb

```diff
@@ -803,8 +803,15 @@ def initialize(options)
       end
       define('--debug', 'Display debug logging') do |_|
         @options[:debug] = true
+        # We don't actually set '--log-level debug' here, but once the options are evaluated by
+        # the config class the end result is the same.
+        @warnings << { msg: "Command line option '--debug' is deprecated, setting '--log-level debug' instead." }
```

Comment:
This phrasing is a little weird. The use of a flag in the message makes me assume it's going to be instructions, but it's a description. Maybe just "[...], set '--log-level debug' instead."